spec/models/solidus_paypal_commerce_platform/payment_method_spec.rb

```diff
@@ -119,6 +119,23 @@ def Struct(data) # rubocop:disable Naming/MethodName
         expect(url.query.split("&")).to include("components=buttons")
       end
     end
+
+    context 'when enable_venmo is false' do
+      let(:order) { instance_double(Spree::Order, checkout_steps: { "foo" => "bar" }) }
```

Comment:
I think that setting `checkout_steps` here would be confusing. Can we remove it?

Reply:
Removing it causes an error from the following line because it's an `instance_double` and not a proper factory.
https://github.com/solidusio-contrib/solidus_paypal_commerce_platform/blob/35524e86421e22258ca3e0081400b17c6bca7c9c/app/models/solidus_paypal_commerce_platform/payment_method.rb#L63